src/ORM/Search/SearchContextForm.php

```diff
@@ -0,0 +1,210 @@
+<?php
+
+namespace SilverStripe\ORM\Search;
+
+use SilverStripe\Control\RequestHandler;
+use SilverStripe\Core\ClassInfo;
+use SilverStripe\Forms\CompositeField;
+use SilverStripe\Forms\FieldList;
+use SilverStripe\Forms\Form;
+use SilverStripe\Forms\GridField\GridField;
+use SilverStripe\Forms\GridField\GridField_FormAction;
+use SilverStripe\Forms\GridField\GridFieldFilterHeader;
+use SilverStripe\ORM\FieldType\DBHTMLText;
+use SilverStripe\View\SSViewer;
+
+/**
+ * A form for using a search context in the CMS.
+ *
+ * Note that the form submission goes through to the index action of the controller by default,
```

Comment:
```suggestion
 * Note that the form submission goes through to the index action of the controller by default,
```
Could you point me to the bit where the controller actually processes the search form submission, I'm not sure where it is

Reply:
See `CMSMain::ListViewForm()` and `GridFieldFilterHeader::getManipulatedData()`